src/content/community/Community-Developers.md

```diff
@@ -0,0 +1,86 @@
+---
+title: GraphQL for Developers
+layout: docs
+category: Community
+permalink: /community/developers/
+next: /community/project-resources/
+sublinks: How GraphQL is developed,Participating in development,GraphQL project governance
+---
+
+The following resources are intended to describe how GraphQL development processes work, how to get involved, and where to get help.
+
+## Essential links
+
+* The [GraphQL Specification](https://spec.graphql.org)
+* GraphQL [reference implementations and tools](/code/)
+* The official [GraphQL GitHub org](https://github.com/graphql)
+* The official [GraphQL Slack](https://slack.graphql.org)
+* Instructions on how to sign the [GraphQL Specification Membership Agreement](https://github.com/graphql/graphql-wg/tree/HEAD/membership)
+* The [GraphQL project calendar](https://calendar.graphql.org)
+* Meeting replays on the [GraphQL YouTube channel](https://youtube.graphql.org)
+
+## How GraphQL is developed
+
+At its core, the GraphQL project is organized around the [specification](https://spec.graphql.org), with a wide variety of supporting implementations and tools.
+
+GraphQL has an active and mutually beneficial relationship with its many implementations. The GraphQL specification is continuously evolving under the care of the [GraphQL Working Group](https://github.com/graphql/graphql-wg), which consists of GraphQL spec experts, contributors to public [reference implementations](/code/), and implementers. At any given time, GraphQL specification updates are a combination of anticipatory planning with documentation of patterns and behaviors that are already proven in production, sometimes at very large scale.
+
+### Working groups
+
+The GraphQL specification, sub-specifications, and official reference implementations are developed using the working group model. The working groups meet monthly or as-needed in order to review ideas and plan their work.
+
+#### The GraphQL Working Group
+
+The [GraphQL Working Group](https://github.com/graphql/graphql-wg) is the main group that plans and implements changes to the [GraphQL specification](https://spec.graphql.org). It meets actively and is the default place for discussion for topics which do not have a separate breakout working group (e.g., input unions, graphql-over-http, etc).
```

Comment:
The `e.g.` here is ambigous; it could imply that these topics don't have separate breakout working groups. I've clarified.

```suggestion
The [GraphQL Working Group](https://github.com/graphql/graphql-wg) is the main group that plans and implements changes to the [GraphQL specification](https://spec.graphql.org). It meets actively and is the default place for discussion for topics which do not have a separate breakout working group (breakout working groups include the [Input Union WG](https://github.com/graphql/graphql-spec/blob/main/rfcs/InputUnion.md) and the [GraphQL-over-HTTP WG](https://github.com/graphql/graphql-over-http)).
```